Gemfile

```diff
@@ -124,6 +124,7 @@ gem 'redis-namespace'
 gem 'request_store'
 gem 'restforce'
 gem 'rgeo-geojson'
+gem 'rswag'
```

Comment:
this was the change missing previously causing things to break in higher environments